website/sidebars.ts

```
@@ -0,0 +1,139 @@
+ - provide next/previous navigation
+
+ The sidebars can be generated from the filesystem, or explicitly defined here.
+
+ Create as many sidebars as you want.
+ */
+
+// For the first build, typedoc-sidebar.js might not exist yet
+let apiSidebar = [];
+try {
+  // Dynamic import for the TypeDoc generated sidebar
+  apiSidebar = require('./docs/api/typedoc-sidebar.js');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)